packages/appstore/src/constants/platform-config.ts

```diff
@@ -5,9 +5,9 @@ import { TAccountCategory, TRegionAvailability } from 'Types';
 
 export type AccountType = { text: 'Real' | 'Demo'; value: TAccountCategory };
 export type RegionAvailability = 'Non-EU' | 'EU' | 'All';
-export const account_types: AccountType[] = [
-    { text: 'Demo', value: 'demo' },
-    { text: 'Real', value: 'real' },
+export const account_types: () => AccountType[] = () => [
```

Comment:
```suggestion
export const getAccountTypes: () => AccountType[] = () => [
```
Since it is a function, now best to use the naming convention for functions